hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -902,7 +902,8 @@ private void checkCellSizeLimit(final HRegion r, final Mutation m) throws IOExce
       while (cells.advance()) {
         int size = PrivateCellUtil.estimatedSerializedSizeOf(cells.current());
         if (size > r.maxCellSize) {
-          String msg = "Cell with size " + size + " exceeds limit of " + r.maxCellSize + " bytes";
+          String msg = "Cell[" + Bytes.toString(m.getRow()) + "] with size " + size
+            + " exceeds limit of " + r.maxCellSize + " bytes";
           LOG.debug(msg);
```

Comment:
Use log4j parameterised messages capabilities?

Reply:
msg will be used in 'throw new DoNotRetryIOException(msg)', so it can not use parameterized messages.

Reply:
Better use toStringBinary?

Reply:
I found that only cells.current() may be better, the KeyValue.toString() method will be called, which print out  rowkey, column, timestamp, type, vlen, and seqid. What do you think @Apache9 ?